build-config/make/mtdutils.make

```diff
@@ -24,7 +24,7 @@ MTDUTILS_STAMP		= $(MTDUTILS_SOURCE_STAMP) \
 			  $(MTDUTILS_BUILD_STAMP) \
 			  $(MTDUTILS_INSTALL_STAMP)
 
-MTDBINS = mkfs.jffs2 mkfs.ubifs ubinize ubiformat ubinfo mtdinfo
+MTDBINS = mkfs.jffs2 mkfs.ubifs ubinize ubiformat ubinfo mtdinfo ubiattach ubimkvol ubidetach ubirmvol
```

Comment:
Are you sure this plays nice with the mtdutils-install target, later in this file?  It just does a blind copy.
ubimkvol starts as a symlink to /bin/busybox, so I wonder if this will over write busybox?

Also if the mtdutils version of the ubi stuff is better and we need it, then can we remove the ubi tools from busybox all together?